### PR TITLE
problem with isSelected 

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by jmau111** [comment]
Hi, I think it's missing, cannot make it work and realize there was a missing dependency.

**Comment by jmau111** [comment]
@youknowriad I've fixed it maybe this is due to the last changes in Gutenberg or a typo in function params.

**Review by ghost — COMMENTED** [review]
`edit({ attributes, setAttributes, isSelected, className }) {` is the right solution to get the select, see 2nd commit.

File: scripts/03-advanced-static-block/index.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ registerBlockType("gew/advanced-static-block", {
     }
   },
 
-  edit({ className, attributes, setAttributes }) {
+  edit({ attributes, setAttributes, isSelected, className }) {
```
Comment (youknowriad) [inline scripts/03-advanced-static-block/index.js]: Good catch, I think there's another way to fix it. isSelected is not usefull anymore to show/hide inspector controls, it's done automatically, we can just drop the test.

Comment (jmau111) [reply scripts/03-advanced-static-block/index.js]: good to know, indeed if it's not useful anymore. Did not know about that 👇

**Review by youknowriad — APPROVED** [review]
Thanks for the fix 👍

**Pull request merged**